lib/simple_app.dart

```diff
@@ -17,9 +17,22 @@ class MyApp extends StatefulWidget {
 
```

Comment:
Is this actually changing the locale of the app now?  cool!

Reply:
I tested it again, and it works perfectly.